src/Player.cpp

```diff
@@ -3214,9 +3225,10 @@ void PLAYER::RobotPump() {
             DebugBreak();
         }
 
-        if ((PlayerNum == 2 && Sim.GetMinute() > 0 && RobotActions[0].ActionId != ACTION_VISITROUTEBOX && ((Sim.GetMinute() / 5) % 2) == 0) ||
-            (RobotUse(ROBOT_USE_ALLRUN) && Sim.GetMinute() > 0 && RobotActions[0].ActionId != ACTION_VISITROUTEBOX)) {
```

Comment:
You removed quite a bit here, could you explain your reasoning? I'm not quite versed in the AI logic, so happy to hear learn more

Reply:
Do you know how "fast-forward" works when the human player ends his day? Works quite different compared to normal game simulation. There is no airport/blocks anymore, no running anywhere, the bots simply count down SpeedCount and then do something, when it reaches 0. This causes numerous problems:
1.) The WalkSpeed (potentially increases by fitness trainer!) was not considered. I noticed it when I saw that my bots gets far fewer things done and performs much worse when you always use "fast-forward" in the morning already. Fixed.
2.) When the bot is running, he is obviously even faster. Therefore, I am dividing SpeedCount by 2. Before this, running was making not difference when fast-forwarding.
3.) Several sabotages do not work anymore when fast-forwarding (poisoning, strike, glue, ...). Not fixed yet. 

Another change here:
There is a weird exception here for Phoenix Travel when played by a bot. He would sometimes run. I removed it so that it is always controlled by the ROBOT_USE_ALLRUN switch.